tasks/17_truncate-string/truncate-string.js

```diff
@@ -0,0 +1,8 @@
+export function truncateString(str, maxLength) {
+  if (str.length > maxLength) {
+    return str.slice(0, maxLength <= 3 ? maxLength : maxLength -3) + '...';
```

Comment:
👍 
but would be better to move `maxLength <= 3 ? maxLength : maxLength -3` to some variable and than pass this variable to slice function